aten/src/ATen/native/Convolution.cpp

```diff
@@ -187,6 +194,146 @@ auto ConvParams::is_depthwise(
          weight.size(0) % input.size(1) == 0; // output channels must be a multiple of input channels
 }
 
+// Check workload to activate fast depthwise FP16 cudnn conv kernels
+bool check_cudnn_depthwise_workload(const at::Tensor& input, const at::Tensor& weight, int stride) {
+  int w = input.size(3);  // same as h
+  int ch = input.size(1);
+  int bs = input.size(0);
+  int k = weight.size(2); // kernel size
```

Comment:
you never use `k` in this function

Reply:
You are right! That's some dead code and I'll remove it.